pkg/helm/chartproxy/testdata/sampleRepoIndex4.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: v1
```

Comment:
for better readability, can we please name this file to reflect the fact that the index contains relative urls, e.g. `sampleRepoIndexWithRelativeURLs.yaml`?